src/import/speed_limiter.rs

```diff
@@ -324,6 +324,7 @@ mod metronome {
         // actual test case inside `catch_unwind`.
         use std::panic;
 
+        let original_hook = panic::take_hook();
```

Comment:
Why not just use `panic_hook::recover_safe()`?

Reply:
Since it generates a panic inside a new thread and checks whether or not that panic will cause problems. For this reason, we cannot use `recover_safe` in the new thread to eat that panic (which is exactly what this test checks).